test/e2e/minor_version_compatibility.go

```diff
@@ -27,14 +32,10 @@ func MinorVersionCompatibility(logger *log.Logger) error {
 	if len(versions) == 0 {
 		logger.Fatal("no versions to test")
 	}
-	numNodes := 4
 	r := rand.New(rand.NewSource(seed))
 	logger.Println("Running minor version compatibility test", "versions", versions)
 
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
-	testNet, err := testnet.New(ctx, "runMinorVersionCompatibility", seed, nil, "test")
+	testNet, err := testnet.New(ctx, "MinorVersionCompatibility", seed, nil, "test")
```

Comment:
Why change the name here?

Reply:
It seems like the term `run` does not seem to add any value to the test name. If it is preferred to keep it, we can put it back.